config/views/template_view_with_context.py

```diff
@@ -7,5 +6,5 @@ class TemplateViewWithContext(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["page_title"] = gettext(self.page_title) if self.page_title else None
+        context["page_title"] = self.page_title
```

Comment:
This might not be equivalent if self.page_title is None; that might be turned into the string "None", I think

Reply:
I think it should be retained as a `None` until it hits the template, which should then be the layer which deals with the problem.

Reply:
Have handled everywhere page_title is used to do a None check